src/components/Switch/Switch.less

```diff
@@ -0,0 +1,130 @@
+@Switch-height: 28px;
+@Switch-width: 50px;
+@Switch-animation-timing: 0.2s ease-in-out;
+@Switch-selected-transform: translate(22px, 0);
+
+@Switch-container-height: 22px;
+@Switch-container-offset: 3px;
+@Switch-container-width: 44px;
+
+@Switch-glow-offset: 5px;
+@Switch-glow-radius: 9px;
+@Switch-glow-scale: scale(1.556);
+
+@Switch-handle-offset: 6px;
+@Switch-handle-radius: 8px;
+@Switch-handle-scale: scale(1.125);
```

Comment:
If we are moving toward using CSS Modules then every file should be responsible for it's own variables.